### DISCUSSION TITLE
Is it possible to update the stored client_secret

### DISCUSSION THREAD

**Discussion by sgabriel** [panva/openid-client#396] [Q&A]
Sign in with Apple requires a dynamically generated client_secret, see this [article](https://developer.okta.com/blog/2019/06/04/what-the-heck-is-sign-in-with-apple).

I'm wondering if it is possible to update it on the existing client instance or if I need to create a new client instance.

**Comment by panva** [comment]
I believe you need to instantiate a new Client instance.

**Reply by sgabriel** [reply]
Thank for the fast answer. I'll do that.

**Reply by dominictobias** [reply]
@sgabriel Some years late but did you figure it out?
